src/viem/Key.ts

```diff
@@ -241,11 +241,14 @@ export async function createWebAuthnP256(
 
   const credential = await WebAuthnP256.createCredential({
     authenticatorSelection: {
-      requireResidentKey: false,
+      requireResidentKey: true,
       residentKey: 'preferred',
```

Comment:
If `requireResidentKey` is truthy, then `residentKey` should also be required (they are the same, but `requireResidentKey` is a "deprecated" field. 
```suggestion
      residentKey: 'required',
```